workflow/apm/services.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: APM Dashboard - Services
+sidebarTitle: Services
+---
+
+The APM Services Dashboard displays all of your APM services in a unified dashboard. A service groups together interconnected endpoints, queries, or jobs within your application. This is useful for contextualizing broader issues within your software infrastructure. 
+
+The dashboard landing page displays common metrics related to your APM services like requests, latency, and errors. Use the Resources, Traces, and Logs tabs to drill into your service-specific data. This dashboard is available across other APM dashboards and the log page to easily understand how data is related to other aspects of your application. 
```

Comment:
I would recommend calling out that clicking on any service will navigate you to a deeper view for that given service, which includes resources, traces, etc.